### ISSUE TITLE
a problem when runing Compute normalized 4C score per fragment

### ISSUE THREAD

**Issue by lwf20** [deLaatLab/pipe4C#33]
Hi, Peter
I made a new bs.genome package with my custom genome and run the pipe4C pipeline. but it stopped in the "Compute normalized 4C score per fragment" step. here is the ERROR message:

>>> Create frag map for genome: rabbit_NZW with RE1: GATC and RE2: CATG and capture length:117 <<<                                        
         ### Loading: /data/linweifeng/4c/fragfolder/rabbit_NZW_GATC_CATG.rds                                                                   
         ### Retrieve and store the unique fragends
         ### Selecting unique fragments from whole genome
      >>> Align reads to fragments <<<
      >>> Compute statistics and create report <<<
      >>> Compute normalized 4C score per fragment <<<
Error in `[[<-`(`*tmp*`, name, value = 0) :
  1 elements in value to replace 0 elements
Calls: Run.4Cpipeline -> norm4C -> $<- -> $<- -> [[<- -> [[<-
Execution halted

Do you know the reason leading to this ERROR?
Looking forward to your reply.

best, weifeng

**Comment by krijgerp** [comment]
Hi weifeng,
Please  share your bam file and data/linweifeng/4c/fragfolder/rabbit_NZW_GATC_CATG.rds (delaatbioinf@hubrecht.eu)
and i will have a look.  Best, Peter

**Comment by lwf20** [comment]
Hi, Peter,
I have sent the rds file and bam files to your email. thank you very much.

**Comment by lwf20** [comment]
Yes, deleting the "chr" of the prefix:chr in the config file. and the pipeline could run successfully .




-----原始邮件-----
发件人:"Johan Zicola" ***@***.***>
发送时间:2024-04-13 01:14:37 (星期六)
收件人: deLaatLab/pipe4C ***@***.***>
抄送: "Weifeng Lin" ***@***.***>, "State change" ***@***.***>
主题: Re: [deLaatLab/pipe4C] a problem when runing Compute normalized 4C score per fragment (Issue #33)






I have the same problem. Did you find what was the issue?

**Comment by johanzi** [comment]
Thanks lwf20 :). Yes, I had the "chr" prefix in both my BSgenome and bowtie2 indexed and therefore added "chr8" in the VPinfo file for my viewpoint, but it should be without prefix (so 8 instead of chr8 in the VPinfo file). Peter Krijger told me he would clarify this point in the documentation. Now it works.
